routes/web.php

```diff
@@ -11,14 +11,27 @@
 |
 */
 
-Route::view('/', 'pages.index')->name('index');
+// INDEX //
+Route::get('/', 'IndexController@index')->name('index');
+Route::get('about', 'IndexController@about')->name('index.about');
+Route::get('policy', 'IndexController@policy')->name('index.policy');
+Route::get('contact', 'IndexController@contact')->name('index.contact');
+Route::get('cookies', 'IndexController@cookies')->name('index.cookies');
+Route::get('profile', 'IndexController@profile')->name('index.profile'); // TODO måske flytte til ProfileController
 
-//Route::get('/new', function () { return view(''); })->name('createSign');
-Route::get('/ask', 'RequestController@index')->name('ask');
-Route::view('/about', 'pages.about')->name('about');
-Route::view('/signs', 'pages.recent')->name('signs');
+// TRANSLATION //
+Route::get('recent', 'TranslationController@recents')->name('translation.recent');
+Route::get('word'.'/{literal}', 'TranslationController@index')->name('translation.signs');
```

Comment:
Hvorfor bruge string concatenate her, hvor du blot kombinere to strings sammen (det kan blot være `'word/{literal}'`)?

Reply:
Rettet